app/uk/gov/hmrc/eusubsidycompliancefrontend/connectors/RetrieveEmailConnector.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.eusubsidycompliancefrontend.connectors
+
+import com.google.inject.{ImplementedBy, Inject, Singleton}
+import play.api.Logger
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.Error
+import uk.gov.hmrc.eusubsidycompliancefrontend.models.types.EORI
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import uk.gov.hmrc.http.HttpReads.Implicits._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@ImplementedBy(classOf[RetrieveEmailConnectorImpl])
+trait RetrieveEmailConnector {
+
+  def retrieveEmailByEORI(eori: EORI)(implicit hc: HeaderCarrier): Future[Either[Error, HttpResponse]]
+
+}
+
+@Singleton
+class RetrieveEmailConnectorImpl @Inject()(http: HttpClient,
+                                           servicesConfig: ServicesConfig
+                                          )(implicit ec: ExecutionContext
+                                          ) extends RetrieveEmailConnector {
+
+  val logger: Logger = Logger(this.getClass)
```

Comment:
Doesn't look like this is being used?

Reply:
yup, will remove 👍

Reply:
[updated](https://github.com/hmrc/eu-subsidy-compliance-frontend/pull/27/commits/1de6e48b8cfe03078a65f8f945dca870b1dc7f91)